crates/holochain_zome_types/src/warrant.rs

```diff
@@ -21,32 +28,41 @@ use crate::signature::Signed;
     derive_more::Deref,
 )]
 pub struct Warrant {
-    /// The self-proving part of the warrant containing evidence of bad behavior
+    /// The self-proving part of the warrant containing evidence of bad behavior.
     #[deref]
     pub proof: WarrantProof,
-    /// The author of the warrant
+    /// The author of the warrant.
     pub author: AgentPubKey,
-    /// Time when the warrant was issued
+    /// Time when the warrant was issued.
     pub timestamp: Timestamp,
+    /// The warranted agent.
+    pub warrantee: AgentPubKey,
```

Comment:
Added the warrantee here. The warrant proof is stored as a blob, so it's not possible to select from the DB by the warranted agent. Adding this field makes that possible and also seems sensible.

Reply:
Nice, I've been wondering why this got added. Sounds good.